src/routes/PaymentCheckPage.tsx

```diff
@@ -123,8 +123,9 @@ export default function PaymentCheckPage() {
     window.location.replace(authorizationUrl);
   };
 
-  // TODO CHK-923
-  const [month, year] = cardData.expDate.split("/");
+  const stDate = "01/".concat(cardData?.expDate || "");
```

Comment:
What does `stDate` stands for? "Standardized date"? "String date"?

Reply:
It's a typo. string date was its original name.